src/reflect/all_test.go

```diff
@@ -4146,30 +4146,33 @@ func TestValuePanic(t *testing.T) {
 
 func shouldPanic(expect string, f func()) {
 	return
-	defer func() {
```

Comment:
@dkegel-fastly did you mean to make this change?

Reply:
Yes.  lint pointed out that the defer never ran.

Reply:
Oh, that's my fault.  `all_test.go` was a copy from upstream with patches to make it run on tinygo + wasi.  Since tinygo doesn't support panic/recover on wasi, I stubbed that out.  We should probably have a wasm-specific check to disable that, and leave a comment for the next person.